src/components/Content/RightPanel/TaskDetails/TaskDetailsTopbar.js

```diff
@@ -78,9 +78,10 @@ class TaskDetailsTopbar extends Component {
             />
           </div>
           <div className="task-discuss-icons-ctn">
-            <div className="discuss-icon-ctn">
-              <FaEllipsisV size={20} />
-            </div>
+            <TaskOptions
+              deleteTask={this.props.deleteTask}
```

Comment:
Could you reduxify `TaskOptions` so we don't need to pass `deleteTask` manually as a prop? I know I did this with `patchTask` some place, but I almost certainly shouldn't have (I forget the exact context).

Reply:
np